1.9/overview/architecture/node-types.md

```diff
@@ -0,0 +1,79 @@
+---
+post_title: Node Types
+nav_title: Node Types
+menu_order: 1
+---
+
+A DC/OS node is a virtual or physical machine on which DC/OS components run. DC/OS nodes are networked together to form a DC/OS cluster.
+
+A DC/OS cluster is composed of three types of nodes: masters, private agents, and public agents.
+
```

Comment:
"Demilitarized Zone" spelled out is going to be equally confusing to anyone who doesn't already know what a DMZ is in the context of computing. We could call it a "perimeter network", which might be more self explanatory, but DMZ is probably more common.